src/index.js

```diff
@@ -22,25 +22,40 @@ import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
 // EAM-480, en-GB locale used in order to have monday as first day of the week
 import { enGB } from "date-fns/locale";
 import { UPDATE_SCANNED_USER } from "./actions/scannedUserActions";
-import AuthWrapper, { tokens, keycloak } from "./AuthWrapper";
+import AuthWrapper, { keycloak, exchangeToken, injectBearerToken } from "./AuthWrapper";
 
 const jss = create(jssPreset());
 
 unregister();
 polyfill();
 
+const getClientID = ({url}) => {
+    if (url?.startsWith(process.env.REACT_APP_EAM_SERVICES_URL)) {
+        return process.env.REACT_APP_KEYCLOAK_EAM_SERVICES_CLIENTID;
+    } else return process.env.REACT_APP_KEYCLOAK_CLIENTID;
+};
+
```

Comment:
There should be a better, more generic way to get the client id instead of a binary check